node/core/chain-selection/src/tree.rs

```diff
@@ -349,7 +348,7 @@ fn add_block(
 
 // Assuming that a block is already imported, accepts the number of the block
 // as well as a list of reversions triggered by the block in ascending order.
```

Comment:
nit: not an issue with your PR:

```suggestion
/// Assuming that a block is already imported, accepts the number of the block
/// as well as a list of reversions triggered by the block in ascending order.
```